packages/dso-toolkit/src/styles/components/_pager.scss

```diff
@@ -2,7 +2,9 @@ $dso-pager-icon-space: $dso-unit / 2;
 $dso-pager-border-width: $dso-button-border-width;
 $dso-pager-height: $u6;
 
-.pager {
+.pager { // DEPRECATED
+  background-color: $brand-danger;
```

Comment:
De vorige keer dat we een compontent deprecieerden (#792, het progressblock) deden we dat net ff anders: https://github.com/dso-toolkit/dso-toolkit/pull/803/files

Waarom nu niet die `$dso-invalid-color`?